scripts/run_e2e_suites_for_cov.sh

```diff
@@ -0,0 +1,38 @@
+
+export COVERAGE_MODE=true
+export LEDGER_SIM=true
+
+rm -rf coverage/e2e
+killall avalanchego
+rm -f /tmp/testKey.pk
```

Comment:
add a TODO to this line, because really the associated test should be improved, so it execution can be repeated without issues